hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -714,6 +714,40 @@
         } // f: data
       ]
     } // R: PAYLOAD_SWAP_DATA
+    { name: "CMD_INFO_EN4B"
+      swaccess: "rw"
+      hwaccess: "hro"
+      desc: '''Opcode for EN4B.
+
+        If the register is active, it affects in flash / passthrough modes.
```

Comment:
nit: instead of `affects` I would probably say `takes effect` here (also in other desc fields below)